src/globemap.ts

```diff
@@ -906,11 +906,10 @@ module powerbi.extensibility.visual {
             }
             this.layout.viewport = options.viewport;
             this.root.css(this.layout.viewportIn);
-            this.zoomContainer.style({
-                "display": this.layout.viewportIn.height > GlobeMap.ZoomControlSettings.height
+            this.zoomContainer.setAttribute("style",
```

Comment:
Does `setAttribute` work in IE 11?

Reply:
https://www.w3schools.com/jsref/met_element_setattribute.asp

Reply:
How i see, it is work in IE 8 and higher